android/app/src/main/kotlin/net/defined/mobile_nebula/APIClient.kt

```diff
@@ -19,10 +19,10 @@ class APIClient(context: Context) {
         return decodeIncomingSite(res.site)
     }
 
-    fun tryUpdate(siteName: String, hostID: String, privateKey: String, counter: Long, trustedKeys: String): IncomingSite? {
```

Comment:
Was there a reason to rename this? tryUpdate was not the name of the API call - it was named this way because it "tried to update" if an update was available. LongPollWait does not update - DoUpdate does. This function does both.